src/ml_gym/validation/validator_factory.py

```diff
@@ -0,0 +1,16 @@
+from typing import Dict, Any
+from ml_gym.validation.nested_cross_validation import NestedCV
+from ml_gym.gym.gym import Gym
+from ml_gym.blueprints.component_factory import ComponentFactory
+
+
+class ValidatorFactory:
+
+    @staticmethod
+    def get_nested_cv(gym: Gym, gs_config: Dict[str, Any], cv_config: Dict[str, Any]):
+        iterator_key = cv_config["NestedCV"]["iterator_key"]
```

Comment:
I couldnt find sanity checks for this so far. This may not be part of this PR, but the YAML config Loader doesnt seem to check wether input is present at all for specific fields. If input is missing this would throw an error at least here. Would also be good to have a data object for that as well, which gives a sophisticated presentation for the YAML input. Sanity checks can than just be implemented via properties.

Reply:
This would be quite complicated to do at this stage, since for every current and future components we would have to implement those checks. So far all is regarded as a dictionary and when building the components there are implicit checks in place when we create the constructables https://github.com/le1nux/mlgym/blob/master/src/ml_gym/blueprints/constructables.py